torch/onnx/symbolic_opset9.py

```diff
@@ -2194,6 +2194,16 @@ def prim_shape(g, self):
     return g.op('Shape', self)
 
 
+def prim_dtype(g, self):
+    dtype = sym_help._try_get_scalar_type(self)
+    dtype = sym_help.cast_pytorch_to_scalar_type[dtype]
```

Comment:
You might not need to have a separate map here. Can you use index of element in scalar_type_to_onnx list?